RxExample/RxExample/Examples/APIWrappers/APIWrappersViewController.swift

```diff
@@ -139,7 +139,7 @@ class APIWrappersViewController: ViewController {
 
         // also test two way binding
         let textValue = Variable("")
-        _ = textField.rx.textInput <-> textValue
+        (textField.rx.textInput <-> textValue).disposed(by: disposeBag)
```

Comment:
I see, then I think those disposables is not being disposed when dismissed. _View controller disposed with 392 resources
assertion failed: Resources weren't cleaned properly, 98 remained, 84 expected_